src/components/card/product-card/ProductCard.jsx

```diff
@@ -25,6 +33,45 @@ const ProductCard = ({ details, loading, cardIsOnPage }) => {
     cardIsOnPage !== ROUTE_HOME ? "w-25" : "w-24"
   }`;
 
+  const handleWishListClick = async () => {
+    if (userState.isUserAuthTokenExist) {
+      const config = {
+        headers: {
+          authorization: userState.userAuthToken,
+        },
+      };
+
+      if (isInWishList) {
+        await axios.delete(`/api/user/wishlist/${_id}`, config);
+
+        wishListDispatch({
```

Comment:
try to do api calls in try catch will be good while catching errors

Reply:
Thanks for suggesting, I have done the changes.